pointer_helpers.go

```diff
@@ -0,0 +1,5 @@
+package linodego
+
+func Pointer[T any](value T) *T {
```

Comment:
Can we have some notes here to explain why and how to use the `Pointer`? Maybe add an example as well.

Reply:
Good suggestion, thanks!